Content.Server/Kitchen/Components/MicrowaveComponent.cs

```diff
@@ -355,26 +346,20 @@ private void Wzhzhzh()
                     return;
                 }
 
-                if (failState == MicrowaveSuccessState.UnwantedForeignObject)
+                AddTemperature(time / 1000.0f);
+
+                if (recipeToCook != null)
                 {
-                    VaporizeReagents();
-                    EjectSolids();
+                    SubtractContents(recipeToCook);
+                    _entities.SpawnEntity(recipeToCook.Result,
+                        _entities.GetComponent<TransformComponent>(Owner).Coordinates);
                 }
                 else
                 {
-                    if (recipeToCook != null)
-                    {
-                        SubtractContents(recipeToCook);
-                        _entities.SpawnEntity(recipeToCook.Result, _entities.GetComponent<TransformComponent>(Owner).Coordinates);
-                    }
-                    else
-                    {
-                        VaporizeReagents();
-                        VaporizeSolids();
-                        _entities.SpawnEntity(_badRecipeName, _entities.GetComponent<TransformComponent>(Owner).Coordinates);
-                    }
+                    EjectSolids();
```

Comment:
Instead of making this an `else -> eject solids`, I'd just always eject solids after subtracting any for a recipe. because currently things like beakers/bowls get left behind in the microwave and have to be manually ejected.